art/Framework/Principal/ProductRetriever.h

```diff
@@ -358,9 +358,9 @@ namespace art {
   ProductRetriever::getView(ViewToken<ELEMENT> const& token,
                             std::vector<ELEMENT const*>& result) const
   {
-    return getView(token.inputTag_.label(),
-                   token.inputTag_.instance(),
-                   token.inputTag_.process(),
+    return getView(token.inputTag().label(),
+                   token.inputTag().instance(),
+                   token.inputTag().process(),
```

Comment:
Perhaps this can be simplified to:

```C++
return getView(token.inputTag(), ...);
```

which simply forwards to the `getView` overload that takes an `art::InputTag`.  Do you mind making the change (even though it's not strictly related to your API change).  Similar comment re. lines 413ff.

Reply:
Good idea... will push the changes ASAP.